### PR TITLE
Adds support for additional containers (sidecars)

### PR DESCRIPTION
**Pull request by JaredDarling**

### CODE REVIEW

**Comment by JaredDarling** [comment]
I have a fix for the tests that I'll submit today.

**Comment by JaredDarling** [comment]
@jnonino Tests fixed. Ready for final review.

File: main.tf [hunk 1]
```diff
@@ -15,56 +15,57 @@ module "ecs-cluster" {
...
-  start_timeout                = var.start_timeout
-  stop_timeout                 = var.stop_timeout
-  privileged                   = var.privileged
-  system_controls              = var.system_controls
-  hostname                     = var.hostname
-  disable_networking           = var.disable_networking
-  interactive                  = var.interactive
-  pseudo_terminal              = var.pseudo_terminal
-  docker_security_options      = var.docker_security_options
+  additional_containers          = var.additional_containers
```
Comment (JaredDarling) [inline main.tf:21]: additional_containers is the new variable. The rest are modified due to re-ordering (sorted).

File: main.tf [hunk 2]
```diff
@@ -115,6 +117,7 @@ module "ecs-fargate-service" {
 
   # Application Load Balancer
   custom_lb_arn                       = var.custom_lb_arn
+  additional_lbs                      = var.additional_lbs
```
Comment (JaredDarling) [inline main.tf:120]: additional_lbs is also a new feature.

File: main.tf [hunk 3]
```diff
@@ -75,19 +76,20 @@ module "td" {
...
-  ecs_cluster_arn                    = module.ecs-cluster.aws_ecs_cluster_cluster_arn
+  deployment_controller              = var.deployment_controller
   deployment_maximum_percent         = var.deployment_maximum_percent
   deployment_minimum_healthy_percent = var.deployment_minimum_healthy_percent
-  deployment_controller              = var.deployment_controller
   desired_count                      = var.desired_count
+  ecs_cluster_arn                    = module.ecs-cluster.aws_ecs_cluster_cluster_arn
   enable_ecs_managed_tags            = var.enable_ecs_managed_tags
   enable_execute_command             = var.enable_execute_command
+  force_new_deployment               = var.force_new_deployment
```
Comment (JaredDarling) [inline main.tf:92]: New feature to keep service always up to date.

**Review by jnonino — APPROVED** [review]

**Pull request merged**